app/models/account.rb

```diff
@@ -56,6 +57,11 @@ def level_set(step)
 
     save
   end
+
+  def hook_req
+    req_params = to_json(only: %i[email level])
+    RestClient.put(ENV['peatio_webhook'].to_s, req_params)
```

Comment:
I think it is better to use ENV.fetch['PEATIO_WEBHOOK_URL']

Reply:
Rewrite according to this review